drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -389,11 +395,11 @@ create_split_virtq(struct vdpa_priv *priv, int index,
 	if (is_virtq_recvq(index, priv->nr_vring)) {
 		MLX5_SET(virtq, virtq, queue_type,
 			 MLX5_VIRTQ_OBJ_QUEUE_TYPE_RX);
-		MLX5_SET(virtq, virtq, qnum, priv->virtq[index].rqn);
+		MLX5_SET(virtq, virtq, qnum, priv->virtq[index].qn);
 	} else {
 		MLX5_SET(virtq, virtq, queue_type,
 			  MLX5_VIRTQ_OBJ_QUEUE_TYPE_TX);
-		/* TODO(liel): Please assign here the SQN to qnum */
+		MLX5_SET(virtq, virtq, qnum, priv->virtq[index].qn);
```

Comment:
Although this code is removed in new PRM, it was nicer to have it out of the if/else as it's the same line

Reply:
right, moved it out of if/else